src/editor/object_option.cpp

```diff
@@ -410,13 +410,15 @@ FileObjectOption::FileObjectOption(const std::string& text, std::string* pointer
                                    std::vector<std::string> filter,
                                    const std::string& basedir,
                                    bool path_relative_to_basedir,
+                                   std::vector<std::string> additional_filter,
```

Comment:
Pass `additional_filter` as const reference.

Reply:
do same for filter?

Reply:
Yes.